domains/zerotwo.json

```diff
@@ -1,11 +1,11 @@
 {
-  "description": "The site for ZeroTwo's docs.",
-  "repo": "https://github.com/ZeroTwoDev",
+  "description": "landing site for zerotwo, the discord bot.",
+  "repo": "https://github.com/ZeroTwoDiscord/website",
   "owner": {
-    "username": "ZeroTwoDev",
-    "email": "halil_ismail@yahoo.com"
+    "username": "Connor",
+    "email": "[redacted] send me a dm on discord instead."
```

Comment:
uh left email an empty string and add an discord pls (discord = Connor!#0800, id 744603004493365330 i suppose)